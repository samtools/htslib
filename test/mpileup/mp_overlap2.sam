@HD	VN:1.5	SO:coordinate
@SQ	SN:1	LN:249250621
r1	99	1	100003	10	2M5D5M	*	0	0	TAACAGA	BBBBBBB
r1	147	1	100003	50	12M	*	0	0	TAAGCACACAGA	ZZZZZZZZZZZZ
