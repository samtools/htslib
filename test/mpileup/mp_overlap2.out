1	100003	2	^+T^St	{!
1	100004	2	A-5()a	{!
1	100005	2	*a	{!
1	100006	2	*g	{!
1	100007	2	*c	{!
1	100008	2	*a	{!
1	100009	2	*c	{!
1	100010	2	Aa	{!
1	100011	2	Cc	{!
1	100012	2	Aa	{!
1	100013	2	Gg	{!
1	100014	2	A$a$	{!
